generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management.module.ts

```diff
@@ -0,0 +1,68 @@
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { RouterModule } from '@angular/router';
+
+import { InfiniteScrollModule } from 'angular2-infinite-scroll';
+
+import { <%= angular2AppName %>SharedModule } from '../../shared';
+<%_ for (var rel of differentRelationships) { 
+       if (rel.otherEntityNameCapitalized === 'User') { _%> 
+import { <%= angular2AppName + 'AdminModule' %> } from '../../admin/admin.module';
```

Comment:
What component are we using from AdminModule? I would like to avoid such dependencies when possible. If something is shared it should be moved to shared module else this will be a recipe for disaster